plasmapy/atomic/tests/test_ionization_state.py

```diff
@@ -294,3 +340,72 @@ def test_execeptions(test):
         kwargs=tests_for_exceptions[test].inputs,
         expected_outcome=tests_for_exceptions[test].expected_exception,
     )
+
+
+kwargs = {
+    'particle': 'He-4',
+    'ionic_fractions': [0.2, 0.3, 0.5],
+    'T_e': 5.0 * u.kK,
+    'tol': 2e-14,
+    'n_elem': 1e13 * u.cm ** -3
+}
+
+expected_properties = {
+    'T_e': 5000.0 * u.K,
+    'tol': 2e-14,
+    'isotope': 'He-4',
+    'element': 'He',
+    'atomic_number': 2,
+    'Z_mean': 1.3,
+    'Z_rms': 1.51657508881031,
+    'n_e': 1.3e19 * u.m ** -3,
+    'n_elem': 1e19 * u.m ** -3,
+    'integer_charges': [0, 1, 2],
+    'ionic_fractions': np.array([0.2, 0.3, 0.5]),
+    'ionic_symbols': ['He-4 0+', 'He-4 1+', 'He-4 2+'],
+    '_is_normalized()': True,
+    'number_densities': np.array([2e18, 3e18, 5e18]) * u.m ** -3,
+    'tol': 2e-14,
+    '__str__()': "<IonizationState of He-4>",
+    '__repr__()': "<IonizationState of He-4>",
+}
+
+instance = IonizationState(**kwargs)
+
+@pytest.mark.parametrize('key', expected_properties.keys())
+def test_IonizationState_attributes(key):
+    """
+    Test a specific case that the IonizationState attributes are working
+    as expected.
+    """
+
+    expected = expected_properties[key]
+    actual = eval(f'instance.{key}')
+
+    if isinstance(expected, u.Quantity):
+        assert expected.unit == actual.unit, \
+            f"Unit mismatch for IonizationState.{key}"
+        assert np.allclose(expected, actual, atol=1e-15 * expected.unit), \
```

Comment:
I can and will keep linking [assert_quantity_allclose](http://docs.astropy.org/en/stable/api/astropy.tests.helper.assert_quantity_allclose.html) for as long as necessary. :laughing:

Reply:
Ah!  I forgot about that.  Good catch!